sn_networking/src/transfers.rs

```diff
@@ -30,8 +28,10 @@ impl Network {
     /// If we get a quorum error, we enable re-try
     pub async fn get_raw_spends(&self, address: SpendAddress) -> Result<Vec<SignedSpend>> {
         let key = NetworkAddress::from_spend_address(address).to_record_key();
+        // use Quorum::One not All here, as we want to collect and return all the spends we can find
+        // Quorum all may be prohibitive to this end
         let get_cfg = GetRecordCfg {
-            get_quorum: Quorum::Majority,
+            get_quorum: Quorum::One,
```

Comment:
I'm not sure what you mean here? Doesn't Quorum::One stop at the first response??

Reply:
old point no longer relevant with early exit removed.